src/messages/manifestjson.js

```diff
@@ -23,6 +23,15 @@ export const MANIFEST_FIELD_INVALID = {
   file: MANIFEST_JSON,
 };
 
+export const MANIFEST_V3_FIREFOX_ANDROID_LIMITATIONS = {
+  code: 'MANIFEST_V3_FIREFOX_ANDROID_LIMITATIONS',
+  message: i18n._(
+    'Firefox for Android Manifest Version 3 is currently limited.'
+  ),
+  description: i18n._('See https://mzl.la/TODO for more information.'),
+  file: MANIFEST_JSON,
+};
```

Comment:
@willdurand I opted for a new rule code with the following rationale:
- the rule code would be specific to mv3 limitations on Firefox for Android
- the rule code would potentially apply to other mv3 limitation other than the one specifically related to the host_permissions and the badge highlighting on Desktop when the extension is pending on the user granting those permissions
- included  "FIREFOX_ANDROID" spelled explicitly in the new rule code because we do have at least one existing rule code including that already

Let me know if we do want to tweak the rule code (or the name or description) to use a different convention and text.

@dotproto to complete this PR we would need to agreed on where to point the shortened url to be included in the final version of this patch, the amount of details we can fit into the rule message or description is pretty limited and would not allow us to provide that much guidance (and the warning by itself is unlikely to be really useful without a doc page where we may direct the developers to get more details about the limitations they will need to consider), would you mind to help me with that part?

Reply:
Link to additional docs moved to the separate followup tracked by #5110